ecosystem/indexer-grpc/indexer-grpc-data-service/src/config.rs

```diff
@@ -59,10 +53,11 @@ pub struct IndexerGrpcDataServiceConfig {
     /// The size of the response channel that response can be buffered.
     #[serde(default = "IndexerGrpcDataServiceConfig::default_data_service_response_channel_size")]
     pub data_service_response_channel_size: usize,
-    /// A list of auth tokens that are allowed to access the service.
+    /// Deprecated: a list of auth tokens that are allowed to access the service.
+    #[serde(skip)]
     pub whitelisted_auth_tokens: Vec<String>,
-    /// If set, don't check for auth tokens.
-    #[serde(default)]
+    /// Deprecated: if set, don't check for auth tokens.
+    #[serde(skip)]
     pub disable_auth_check: bool,
```

Comment:
Why don't we just delete these from the config? Are we keeping it here for local testing?

Reply:
community requires to change the config, otherwise, they'll break.